ergotree-ir/src/types/stype.rs

```diff
@@ -56,6 +56,12 @@ pub enum SType {
     SFunc(SFunc),
     /// Context object ("CONTEXT" in ErgoScript)
     SContext,
+    /// Header of a block
+    SHeader,
+    /// Header o block without solved mining puzzle
+    SPreHeader,
+    /// Data type introduced to unify handling of global and non-global (i.e. methods) operations.
+    SGlobal,
```

Comment:
Please, add new `SType` variants to the `Arbitrary` impl. It'll cover type serialization code.